tests/mock/Ed25519Signature2018.js

```diff
@@ -78,6 +78,16 @@ mock.parameters.sign = {
   })
 };
 
+// this links back to a key with an assertionMethod on it
```

Comment:
What does "a key with an assertionMethod on it" mean? This comment needs adjusting (and the same comment appears in RsaSignature2018.js). Should this say "a key with a controller that is an object"?

Reply:
Changed to `key with a controller object` https://github.com/digitalbazaar/jsonld-signatures/commit/c7f463c7169addf6bcdb0dec6f35f64b7994a3ac